paymentsheet/src/test/java/com/stripe/android/paymentsheet/model/SupportedPaymentMethodTest.kt

```diff
@@ -24,11 +24,12 @@ import java.io.File
 @RunWith(AndroidJUnit4::class)
 class SupportedPaymentMethodTest {
     private val lpmRepository = LpmRepository(
-        LpmRepository.LpmRepositoryArguments(
-            ApplicationProvider.getApplicationContext<Application>().resources
+        arguments = LpmRepository.LpmRepositoryArguments(
+            resources = ApplicationProvider.getApplicationContext<Application>().resources,
+            isUpiEnabled = true
         )
     ).apply {
-        this.forceUpdate(listOf(PaymentMethod.Type.Card.code, PaymentMethod.Type.Eps.code), null)
+        this.forceUpdate(this.supportedPaymentMethods, null)
```

Comment:
By only passing in `listOf(PaymentMethod.Type.Card.code, PaymentMethod.Type.Eps.code)`, I believe we have only ever tested these two. We should test all supported payment methods.